TOOLS/ce/src/main/scala/org.opalj.ce/ce.scala

```diff
@@ -0,0 +1,65 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+import java.io.File
+import java.nio.file.Paths
+import scala.collection.mutable.ListBuffer
+
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+
+/*
+ *   Standalone configuration explorer
+ *   This is the main method that runs the configuration explorer
+ *   It creates a browsable HTML File out of the configuration files present in the entire OPAL project
+ */
+object ce extends App {
```

Comment:
Upper case class/object names and don't use abbreviations.

Reply:
Fixed